src/main/java/org/folio/circulation/resources/RequestNoticeSender.java

```diff
@@ -72,9 +72,7 @@ public RequestNoticeSender(PatronNoticeService patronNoticeService,
     patronNoticeService.acceptNoticeEvent(requestCreatedEvent, NoticeLogContext.from(request));
 
     Loan loan = request.getLoan();
-    if (request.getRequestType() == RequestType.RECALL &&
-      loan != null && loan.hasDueDateChanged()) {
-
+    if (request.getRequestType() == RequestType.RECALL && loan != null) {
```

Comment:
Should this logic have also changed when a request is moved (the method below)? 

Scenario 5 in the issue suggests so to me.

Reply:
Thanks @marc-johnson, I will create another PR with this fix

Reply:
It will be fixed in scope of this story: https://issues.folio.org/browse/CIRC-1031